From: Paul Boddie <paul@boddie.org.uk>
Date: Tue, 15 Aug 2023 23:50:33 +0200
Subject: Remove setuptools_scm retrieval

Since the python3-setuptools-scm package is a build dependency, setup.py
should not attempt to retrieve the package from the Python Package Index
itself.

moin (2.0+git20230330.c55318e3-1) unstable; urgency=medium

  * Initial release.

Last-Updated: 2024-02-23
Reviwed-By: Kathara Sasikumar <katharasasikumar007@gmail.com>

Index: moin/setup.py
===================================================================
--- moin.orig/setup.py
+++ moin/setup.py
@@ -69,9 +69,6 @@ setup_args = dict(
     use_scm_version={
         'write_to': os.path.join('src', 'moin', '_version.py'),
     },
-    setup_requires=[
-        'setuptools_scm',  # magically cares for version and packaged files
-    ],
     install_requires=[
         'Babel>=2.10.0',  # internationalization support
         'blinker>=1.5',  # event signalling (e.g. for change notification trigger)
